src/Pretzel.Logic/Configuration.cs

```diff
@@ -54,16 +54,28 @@ private void CheckDefaultConfig()
             {
                 _config.Add("date", "2012-01-01");
             }
+            if (!_config.ContainsKey("configuration"))
+            {
+                _config.Add("configuration", "debug");
+            }
         }
 
-        internal void ReadFromFile()
+        internal void ReadFromFile(IDictionary<string, string> defaults = null)
         {
             _config = new Dictionary<string, object>();
             if (_fileSystem.File.Exists(_configFilePath))
             {
                 _config = _fileSystem.File.ReadAllText(_configFilePath).ParseYaml();
                 CheckDefaultConfig();
             }
+
+            if (defaults != null)
+            {
+                foreach (var key in defaults)
+                {
+                    _config[key.Key] = key.Value;
```

Comment:
I don't think that's a good thing that the default overwrite existing configuration.